src/formats/geotiff/GeoTiffReader.js

```diff
@@ -53,45 +53,32 @@ define([
          * @alias GeoTiffReader
          * @constructor
          * @classdesc Parses a geotiff and creates an image or an elevation array representing its contents.
-         * @param {String} url The location of the geotiff.
+         * @param {String} ArrayBuffer The ArrayBuffer of the GeoTiff
          * @throws {ArgumentError} If the specified URL is null or undefined.
          */
-        var GeoTiffReader = function (url) {
-            if (!url) {
+        var GeoTiffReader = function (arrayBuffer) {
+            if (!arrayBuffer) {
                 throw new ArgumentError(
-                    Logger.logMessage(Logger.LEVEL_SEVERE, "GeoTiffReader", "constructor", "missingUrl"));
+                    Logger.logMessage(Logger.LEVEL_SEVERE, "GeoTiffReader", "constructor", "missing ArrayBuffer"));
```

Comment:
My understanding is that "missingArrayBuffer" should be one word and match a key in the messageTable in Logger.js. I assume this is done for translation purposes, I think we're missing a lot of keys.

Reply:
Our early convention was to create keys for common messages, and put one-off messages inline. So both patterns are valid.

The arrayBuffer argument name is likely to come up again, so I think Mark's recommendation should be implemented.

Reply:
@zglueck Hi Zach, I think the vote was to go with missingArrayBuffer (no space) here.